apps/src/sites/studio/pages/teacher_dashboard/index.js

```diff
@@ -1,14 +1,22 @@
 import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {BrowserRouter as Router, Route, withRouter} from 'react-router-dom';
+import {BrowserRouter as Router, Route} from 'react-router-dom';
 import TeacherDashboard from '@cdo/apps/templates/teacherDashboard/TeacherDashboard';
 
+const script = document.querySelector('script[data-dashboard]');
+const scriptData = JSON.parse(script.dataset.dashboard);
+const sectionId = scriptData.section_id;
```

Comment:
i wonder if the section id is something we could (or should) get from redux instead of the document. this isn't a necessary refactor right now, but it's something i've been thinking about. i'd like to make redux our source of truth for react components, but, as long as we standardize, i don't really care what the source of truth is

Reply:
+1 for section id eventually living in redux, especially because we'll eventually want to have react router modify the section id so that the section switcher doesn't trigger page reloads. Fine to do it this way for now though.